app/services/required_document_category_analyser.rb

```diff
@@ -24,6 +24,13 @@ def call
       required_document_categories << "expert_report"
     end
     required_document_categories << "parental_responsibility" if has_parental_responsibility?
+    required_document_categories << "local_authority_assessment" if local_authority_assessed?
+    required_document_categories << "court_order" if @application.plf_court_order?
+    if @application.public_law_family_proceedings?
+      required_document_categories << "grounds_of_appeal"
+      required_document_categories << "counsel_opinion"
+      required_document_categories << "judgement"
```

Comment:
nit: I added line breaks between multiline breaks - rubocop can be configured to enforce this but I guess we do not bother with this so not sure if that was a conscious decision or not but readability is helped by line breaks IMO